paper/paper.md

```diff
@@ -0,0 +1,75 @@
+
+The development of `xclim` started at Ouranos in 2018 out of the need to deliver data for a pan-Canadian atlas of climate indicators. In-house specialists had different implementations for the same indicators, and there was a desire to adopt a common library that would tie together investments in research and development with operational production capabilities. At the time, the library closest that best met these requirements was `icclim` [@icclim], a library developed in the context of the [European Climate Assessment & Dataset](https://www.ecad.eu/) project, and meant to monitor and analyze changes in climate extremes. It was, however, not designed to be easily extensible, and we believed the indicators they offered could be written more succinctly and computed more efficiently by relying on objects and primitives from `xarray` [@Hoyer:2017]. `xclim` started as a reimplementation of `icclim` with an `xarray` backend, drawing inspiration from `MetPy` [@metpy], and eventually grew to include other algorithms routinely used in climate data analysis.
+
+`xclim` is meant to be one component in a larger software ecosystem for climate data analysis. Other libraries often used in tandem with `xclim` are `clisops` [@clisops] for spatial subsetting and averaging utilities, and `xESMF` [@xesmf] for spatial regridding.
+
+# Key Features
+
+## Climate indicators calculations
+
+An `Indicator` class is built around a `compute` function defining a climate indicator. It performs health checks on input data (units, time frequency, outlier detection), handles missing values, and assigns attributes to the output, so it complies with the Climate and Forecast (CF) Convention [@Hassel:2017]. Indicators can be customized using a context manager, class inheritance, or through a YAML file - the latter allowing the creation of custom collections of indicators for batch processing.
+
+## Statistical adjustment and bias correction
+
+The `xclim.sdba` subpackage provides different algorithms to adjust the distribution of simulated variables to observed variables. It adopts a train / adjust paradigm, where corrections are first calculated, then applied to the target data or saved for later use. Most methods support additive or multiplicative corrections, different time groupings (seasonal, monthly, or daily with a rolling window). Correction factors can be interpolated between time groupings to avoid discontinuities in the corrected data.
+
+## Ensemble analysis
+
+The `xclim.ensembles` subpackage bundles utilities to facilitate the analysis of results from multiple models. It includes functions to reduce the ensemble size using clustering algorithms, metrics of ensemble robustness, and significance of climate change signals.
+
```

Comment:
Should we mention:
-  `analogs`?
- `dataflags` ?
- the `cli` ?

Reply:
I think we should. We don't need to say much. Maybe two sentences each, max.